org-roam.el

```diff
@@ -870,6 +870,26 @@ whose title is 'Index'."
         (concat (expand-file-name org-roam-directory) path)
       index)))
 
+;;;; dealing with file-wide properties
+(defun org-roam--set-global-prop (name value)
```

Comment:
I actually have an idea how we can remove this function in the future. Since 9.4 is out, and files also have properties, we can gradually migrate to storing `ROAM_xxx` as properties (aka `:ROAM_xxx:`) instead of 'settings' (aka `#+ROAM_xxx`).

Reply:
Yeah, this is fine, I think we can file this as a feature request (supporting fetching from property drawers).